test/extended/networking/livemigration.go

```diff
@@ -104,7 +105,7 @@ var _ = Describe("[sig-network][OCPFeatureGate:PersistentIPsForVirtualization][F
 						}{
 							VMName:                    vmName,
 							VMNamespace:               f.Namespace.Name,
-							FedoraContainterDiskImage: "quay.io/kubevirt/fedora-with-test-tooling-container-disk:20241024_891122a6fc",
+							FedoraContainterDiskImage: image.LocationFor("quay.io/kubevirt/fedora-with-test-tooling-container-disk:20241024_891122a6fc"),
```

Comment:
can we track a "release" version instead ? I guess this is the SHA of the release img, but still ...

Reply:
We have to follow kubevirtci release cycle in case we need to modify to add something, so we don't depend on kubevirt cycle.